README.md

```diff
@@ -29,6 +29,62 @@ juju relate ranger-k8s:db postgresql-k8s:database
 ```
 Refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for details on building the charm and ranger image. 
 
+### Group management with Apache Ranger
+The Charmed Ranger Operator makes use of the [Ranger API](https://ranger.apache.org/apidocs/index.html) and [apache-ranger PyPi package](https://pypi.org/project/apache-ranger/) to manage users and groups. The source of users and group memberships is a `user-group-configuration.yaml` file provided to the charm as a configuration value `user-group-configuration`. 
+
+An example of this file is here: 
+```
+ranger-k8s:
+   user-group-configuration: |
+      relation_2:
```

Comment:
Hmm, so that would be really nice but I'm not sure how to achieve it. I came to this method of adding the relation_id in the config, because 1 ranger charm could be related to (for example) 2 trino charms and a kafka charm. In this case we still want to manage users and groups from a central location as users and groups are not service specific in ranger. However, for setting up the unix users in the related application, it makes sense to only share the relevant subset of users specific to that application. Trino doesn't need to know about Kafka users. When connecting an application to Ranger the related application provides a service name. This defaults to relation_id if none is provided, but could be any string. We could use this service name instead? But then it means that you could not manually change the service name in the UI (which is unlikely). But I'm not sure how many pros it has. Interested to hear your thoughts on this!

Reply:
I am not sure I understand what you mean when you say that the we can change the service name in the UI. But, maybe if the service name is smth that changes dynamically, we could use another field which is constant, like `service type` or smth like that (that could be only trino|kafka|whatever).

Reply:
I think the problem here is you have have multiple trino services connected. Also I'm doing a major refactor of this code to match what was discussed on the trino side, so please could you hold off further review!

Reply:
I have made an update which addresses this. It uses the `ranger-service-name` value which is configured on the related application.